src/bootstrap/src/core/build_steps/check.rs

```diff
@@ -839,3 +839,9 @@ tool_check_step!(Linkchecker {
     mode: |_builder| Mode::ToolBootstrap,
     default: false
 });
+
+tool_check_step!(BumpStage0 {
+    path: "src/tools/bump-stage0",
+    mode: |_builder| Mode::ToolBootstrap,
+    default: false
```

Comment:
It would be nice to have the option to check these tools by default on CI, so that we don't have to enumerate them manually in CI scripts.